dbt/adapters/dremio/connections.py

```diff
@@ -254,10 +256,10 @@ def _create_folders(self, database, schema, api_parameters):
             except DremioAlreadyExistsException:
                 logger.debug(f"Folder {folder} already exists.")
             except DremioBadRequestException as e:
-               if "Can not create a folder inside a [SOURCE]" in e.message:
-                   logger.debug(f"Ignoring {e}")
-               else:
-                   raise e
+                if "Can not create a folder inside a [SOURCE]" in e.message:
+                    logger.debug(f"Ignoring {e}")
+                else:
+                    raise e
```

Comment:
[NIT]
revert (why does this PR include so many formatting changes? I would keep this in a separate change)

Reply:
python is about the tabs(or spaces) to separate internal parts from the outer. I see that in this line it's not a problem (maybe before it was not working)